docs/configuration/alarms/discord.rst

```diff
@@ -193,6 +193,15 @@ Example: Alarm Configuration Using Optional Parameters
           "title":"This Pokestop has been invaded by Team Rocket!",
           "url":"<gmaps>",
           "body":"Invasion will expire at <24h_time> (<time_left>)"
+      },
+      "quests":{
+          "webhook_url":"DISCORD_WEBHOOK_URL_FOR_INVASION_CHANNEL",
+          "username":"Quest",
+          "icon_url*":"<YOUR CUSTOM URL HERE>/<quest_image>.png",
```

Comment:
We should be consistent with values we want the user to replace - also consider avoiding `<>` as that currently signals a DTS. 
```suggestion
          "icon_url*":"<YOUR_CUSTOM_URL_HERE>/<quest_image>.png",
```